src/inc/msquic.h

```diff
@@ -295,6 +295,34 @@ typedef struct QUIC_LISTENER_STATISTICS {
     } Binding;
 } QUIC_LISTENER_STATISTICS;
 
+typedef enum QUIC_PERFORMANCE_COUNTERS {
+    QUIC_PERF_COUNTER_CONN_CREATED,         // Total connections ever allocated.
+    QUIC_PERF_COUNTER_CONN_HANDSHAKE_FAIL,  // Total connections that failed during handshake.
+    QUIC_PERF_COUNTER_CONN_APP_REJECT,      // Total connections rejected by the application.
+    QUIC_PERF_COUNTER_CONN_RESUMED,         // Total connections resumed.
+    QUIC_PERF_COUNTER_CONN_ACTIVE,          // Connections currently allocated.
+    QUIC_PERF_COUNTER_CONN_CONNECTED,       // Connections currently in the connected state.
+    QUIC_PERF_COUNTER_CONN_CLOSED_ERROR,    // Total connections closed with a protocol error.
```

Comment:
As I mentioned elsewhere, let's use the word shutdown instead of close (since that's the event generated that has the error).

Reply:
I think the above comment suggested QUIC_PERF_COUNTER_CONN_PROTOCOL_ERRORS. I'm going with this unless there's objections.